❤️ hello world 78783

/Users/apple/Documents/github_repos/learn-express/learn-node/watch-and-recover-process/app.js:4
    throw "❌Oops... Something bad happened for something good!";
    ^
❌Oops... Something bad happened for something good!
(Use `node --trace-uncaught ...` to show where the exception was thrown)

Node.js v22.13.0
❤️ hello world 78891

/Users/apple/Documents/github_repos/learn-express/learn-node/watch-and-recover-process/app.js:4
    throw "❌Oops... Something bad happened for something good!";
    ^
❌Oops... Something bad happened for something good!
(Use `node --trace-uncaught ...` to show where the exception was thrown)

Node.js v22.13.0
❤️ hello world 79026

/Users/apple/Documents/github_repos/learn-express/learn-node/watch-and-recover-process/app.js:4
    throw "❌Oops... Something bad happened for something good!";
    ^
❌Oops... Something bad happened for something good!
(Use `node --trace-uncaught ...` to show where the exception was thrown)

Node.js v22.13.0
❤️ hello world 79137

/Users/apple/Documents/github_repos/learn-express/learn-node/watch-and-recover-process/app.js:4
    throw "❌Oops... Something bad happened for something good!";
    ^
❌Oops... Something bad happened for something good!
(Use `node --trace-uncaught ...` to show where the exception was thrown)

Node.js v22.13.0
❤️ hello world 79245

/Users/apple/Documents/github_repos/learn-express/learn-node/watch-and-recover-process/app.js:4
    throw "❌Oops... Something bad happened for something good!";
    ^
❌Oops... Something bad happened for something good!
(Use `node --trace-uncaught ...` to show where the exception was thrown)

Node.js v22.13.0
❤️ hello world 79357
